draft-packages/pagination/KaizenDraft/PageIndicator.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import cx from "classnames"
+import styles from "./styles.scss"
+
+type PageIndicatorProps = {
+  page: number
+  selected: boolean
+  ariaLabelPage?: string
+  onPageClick: (pageClicked: number) => void
+}
+
+const PageIndicator = ({
+  page,
+  selected,
+  ariaLabelPage,
+  onPageClick,
+}: PageIndicatorProps) => (
+  <button
+    key={`page-indicator-${page}`}
+    className={cx(styles.pageIndicator, {
+      [styles.pageIndicatorSelected]: selected,
+    })}
+    aria-label={`${ariaLabelPage} ${page}`}
```

Comment:
I don't know how important this is, but `ariaLabelPage` is optional, and used here as if it isn't. It's possible that this template literal might spit out `undefined 1`, `null 2` etc.
```suggestion
    aria-label={`${ariaLabelPage || "Page"} ${page}`}
```